<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="title">
     <item row="0" column="0">
      <widget class="QLabel" name="milanTitle">
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 239, 255);
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Milan's Recipe Finder&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="searchBox">
     <item>
      <widget class="QToolButton" name="dropDownBox">
       <property name="text">
        <string>Aditional Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="appOutput"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search üîçÔ∏é</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="timeSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>521</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(143, 143, 143);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timeDisplay">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>No Preference</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>160</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="difficultyCollection">
     <item>
      <widget class="QLabel" name="difficultyLabel">
       <property name="text">
        <string>Difficulty</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="difficultyEasy">
       <property name="text">
        <string>Easy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="difficultyMedium">
       <property name="text">
        <string>Medium</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="difficultyHard">
       <property name="text">
        <string>Hard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="difficultyAny">
       <property name="text">
        <string>Any</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>200</y>
      <width>160</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="allergiesCollection">
     <item>
      <widget class="QLabel" name="allergyLabel">
       <property name="text">
        <string>Allergies</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="nuts">
       <property name="text">
        <string>Nuts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="milk">
       <property name="text">
        <string>Milk</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fish">
       <property name="text">
        <string>Fish</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wheat">
       <property name="text">
        <string>Wheat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="resultList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>761</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionApplication"/>
    <addaction name="separator"/>
    <addaction name="actionLinkedIn"/>
    <addaction name="actionGitHub"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/images/img/icons8-emergency-exit-48.png</normaloff>:/images/img/icons8-emergency-exit-48.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Q</string>
   </property>
  </action>
  <action name="actionApplication">
   <property name="text">
    <string>Application</string>
   </property>
  </action>
  <action name="actionLinkedIn">
   <property name="text">
    <string>LinkedIn</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
